### ISSUE TITLE
redirect from login.php

### ISSUE THREAD

**Issue by aarondiel** [pi-hole/web#2352]
### Versions
Pi-hole version is v5.12 (Latest: v5.12)
AdminLTE version is v5.14.2 (Latest: v5.14.2)
FTL version is v5.17 (Latest: v5.17)

### Platform
running on the latest docker container

### Explenation
i have set up pihole to be served under localhost/pihole via nginx as a reverse proxy, the login page (`/login.php`) simply redirects to /admin, while i need it to redirect to /pihole/admin.

i think this should be a simple bug to fix, but i am not well versed in php シ

### Steps to reproduce
docker-compose.yaml
```yaml
services:
  nginx:
    image: nginx:1.23
    container_name: "nginx"
    hostname: "nginx"
    restart: unless-stopped
    volumes:
      - /root/nginx.conf:/etc/nginx/nginx.conf:ro
      - /etc/letsencrypt:/etc/letsencrypt

    ports:
      - 80:80 # http
      - 443:443 # https

  pihole:
    image: pihole/pihole:latest
    container_name: pihole
    hostname: pihole
    restart: unless-stopped
    ports:
      - 53:53/tcp
      - 53:53/udp
      - 8485:80

    environment:
      TZ: "Europe/Berlin"

    volumes:
      - /root/pihole/etc:/etc/pihole
      - /root/pihole/dnsmasq:/etc/dnsmasq.d
```

nginx.conf:
```nginx
worker_processes 1;

events {
	worker_connections 256;
}

http {
	include mime.types;
	default_type application/octet-stream;
	types_hash_max_size 4096;
	gzip on;

	server {
		listen 443 ssl;
		server_name test.server.com;
		ssl_prefer_server_ciphers on;
		ssl_certificate /etc/letsencrypt/live/test.server.com/fullchain.pem;
		ssl_certificate_key /etc/letsencrypt/live/test.server.com/privkey.pem;

		location / {
			root /usr/share/nginx/html;
			index index.html;
		}

		location /pihole/ {
			proxy_set_header Host $host;
			proxy_set_header X-Real-IP $remote_addr;
			proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
			proxy_set_header X-Forwarded-Proto $scheme;

			proxy_pass http://pihole:80/admin/;

			proxy_read_timeout 90;
			proxy_send_timeout 90;
		}
	}

	server {
		listen 80;
		server_name test.server.com;
		return 301 https://$server_name$request_uri;
	}
}
```

### Screenshots
/pihole before authentication -> login.php
![before_navigation](https://user-images.githubusercontent.com/38919146/190158409-a63986ae-3798-409e-99f9-32adebb2c38c.png)

after authentication, redirected to /admin -> 404 not found
![after_navigation](https://user-images.githubusercontent.com/38919146/190158610-41a9a218-28e5-4aa2-b828-d4c6f46bc952.png)

/pihole while being authenticated is shown correctly
![reroute](https://user-images.githubusercontent.com/38919146/190158989-b145ae37-f7f3-465f-a939-c943a373ea97.png)

### Additional context
this might be an issue with my nginx.conf, but even if it is, i would appreciate if the docs explained how you could use pihole with a reverse proxy.

**Comment by rdwebdesign** [comment]
Yes, looks like your `nginx` is not correctly configured. 

There are too little information to understand what is happening exactly on your system, but looks like you changed the path of your web interface.

Usually the web root is at `/var/www/html/`.
The web interface is installed at `/var/www/html/admin/` and there are some files (the old blocking page, for example) at `/var/www/html/pihole/`.

I'm not used to `nginx` config, but try to change the `location /pihole/` config to `location /`.

**Comment by aarondiel** [comment]
i sadly cannot host pihole on `/` as there is already a different webserver present on that route.

everything under `location /` is just for displaying the default *welcome to nginx* site as a placeholder, nginx is not used to resolve any files or as a replacement for lighttpd.

i did something similar for *syncthing* as it was already described in the [docs](https://docs.syncthing.net/users/reverseproxy.html).

basically: i wanted to to host pihole under `/pihole` instead of `/admin` and it works apart from this one redirect to `/admin` when i log in.

if this is not relevant, feel free to close this issue

**Comment by rdwebdesign** [comment]
Well, I can't help you with your config. As I said I'm not very used to `nginx` config.
Also, `nginx` is not the default webserver.

Since this is a custom setup and there are no bugs on the code, I will close this issue.

I suggest you to ask your questions about configuration at our [Discourse Forum](https://discourse.pi-hole.net/).
Thank you for your understanding.
